Octokit.Reactive/Clients/IObservableIssueTimelineClient.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Octokit.Reactive
+{
+    /// <summary>
+    /// A client for GitHub's Issue Timeline API.
+    /// </summary>
+    /// <remarks>
+    /// See the <a href="http://developer.github.com/v3/issues/timeline/">Issue Timeline API documentation</a> for more information.
```

Comment:
Minor nitpick but the doc comments added in this PR have a mix of http and https on the URLs... 😉